scss/_variables.scss

```diff
@@ -2315,6 +2315,36 @@ $footer-terms-padding-y-md:               $spacer * 1.1 !default;
 $footer-gap:                              $spacer * .75 !default;
 $footer-gap-xl:                           $spacer * 1.7 !default;
 // scss-docs-end footer
+
+//// Side navigation
+// scss-docs-start side-nav-variables
+$side-nav-min-width:                calc(var(--#{$prefix}side-nav-item-icon-size) + var(--#{$prefix}side-nav-item-spacing) * 2 - var(--#{$prefix}side-nav-border-width)) !default; /* stylelint-disable-line function-disallowed-list */
+$side-nav-max-width:                $offcanvas-horizontal-width !default;
+$side-nav-bg:                       var(--#{$prefix}body-bg) !default;
+$side-nav-border-width:             1px !default;
+$side-nav-border-color:             var(--#{$prefix}border-color-translucent) !default;
+
+$side-nav-item-min-height:          $spacer * 2 !default;
+$side-nav-item-icon-size:           $spacer * 1.25 !default;
+$side-nav-item-spacing:             15px !default;
+$side-nav-item-padding-top:         8px !default;
+$side-nav-item-padding-end:         0 !default;
+$side-nav-item-padding-bottom:      10px !default;
+$side-nav-item-padding-start:       15px !default;
```

Comment:
Is there a reason why some values are hardcoded in `px` instead of `$spacer * x`?

Reply:
Yes, IMO, some values doesn't need to grow depending of the font-size compared to others. I think that icons for example should grow or the list-group-item size but not necessarily the padding inside this item. Feel free to debate this pov 😄